codepoints.json

```diff
@@ -241,7 +241,6 @@
     "trash": 61826,
     "heading": 61710,
     "insert": 61711,
-    "saved": 61715,
```

Comment:
This was stored in the codepoints file, but not used as far as I can tell. Each name in this code point is supposed to point to a file, i.e. "saved" means it was pointing to "saved.svg" in the sources/svg folder, but no such file ships in the current version. That file is stored in the sources/svg/gutenberg folder, which in this PR has been removed.